### ISSUE TITLE
GetFeatureInfo working well only to the first layer in the list

### ISSUE THREAD

**Issue by francbartoli** [geobeyond/mapreacter#11]
If you have an array of layers:

`layers: ['nnb:param_oss_point', 'nnb:param_oss_polig', 'nnb:param_dist_geom']`

and the user click on a feature in the map from the second or the third element of the array then the result is empty

**Comment by milafrerichs** [comment]
yes, the display of the WMSPopup needs to reflect the different layers.
they are already in the `items` list but not displayed yet.
